fix include error for RingDecomposerLib.h in case location to RingDecomposerLib sources are specified via -DURFLIB_DIR

author: Kenneth Hoste (HPC-UGent)

--- rdkit-Release_2019_09_1/CMakeLists.txt.orig	2019-11-25 16:44:26.415914160 +0100
+++ rdkit-Release_2019_09_1/CMakeLists.txt	2019-11-25 16:45:08.872102105 +0100
@@ -167,7 +167,11 @@

 if(RDK_USE_URF)
   ADD_DEFINITIONS("-DRDK_USE_URF")
-  include_directories(${RDKit_ExternalDir}/RingFamilies/RingDecomposerLib/src/RingDecomposerLib)
+  if(DEFINED URFLIB_DIR)
+      include_directories(${URFLIB_DIR})
+  else()
+      include_directories(${RDKit_ExternalDir}/RingFamilies/RingDecomposerLib/src/RingDecomposerLib)
+  endif()
 endif()


